ddtrace/appsec/_deduplications.py

```diff
@@ -32,7 +32,7 @@ def _check_deduplication(self):
         return asm_config._asm_deduplication_enabled
 
     def __call__(self, *args, **kwargs):
-        result = None
+        result = False
```

Comment:
Why the change ?

Reply:
Because all the functions where deduplication is applied return a boolean. With this change, we ensure the return type is always `bool`, not `Optional[bool]`.